libs/etterlatte-ktor/src/main/kotlin/initialisering/EmbeddedServerConfig.kt

```diff
@@ -51,6 +55,11 @@ private fun settOppEmbeddedServer(
                 module {
                     body()
                 }
+                module {
+                    environment.monitor.subscribe(ServerReady) {
```

Comment:
Skulle man kanskje få sendt det inn som en modul i steden for?

Reply:
Noen tester som feiler dog

Reply:
Kanskje, men eg har eigentleg lyst til å abstrahere bort heile module-opplegget frå applikasjonane, sånn at det kun er her. Særleg fordi subscribe som gjerast her er noko vi har gjort på mange forskjellige måtar i anna oppstarts-logikk, så viss vi kun kan ha det akkurat her, aukar det sjansen for at vi er konsekvente.

Men synes ikkje dette opplegget med å sende inn map vart heilt fint. Vil heller berre sende inn liste over timer-jobbar, og så kan denne ta seg av resten. Ser litt meir på akkurat det.